components/browser/engine-gecko-nightly/src/test/java/mozilla/components/browser/engine/gecko/selection/GeckoSelectionActionDelegateTest.kt

```diff
@@ -55,6 +57,25 @@ class GeckoSelectionActionDelegateTest {
             Assert.assertTrue(actualActions.contains(it))
         }
     }
+
+    @Test
+    fun `Security exception in perform action will not cause a crash`() {
```

Comment:
nit: Maybe we could use [Given/When/Then](https://martinfowler.com/bliki/GivenWhenThen.html) naming convention

Reply:
Sure, I'll update the test name.  Thanks